subtests/docker_cli/systemd_in_container/systemd_in_container.py

```diff
@@ -0,0 +1,50 @@
+r"""
```

Comment:
Something to think about:

This test (overall) seems much more related to confirming container-content and possibly behavior/interactions with the host.  I'm wondering if maybe it shouldn't get it's own subtest category entirely, so it/they are distinct from ``docker_cli`` specific tests.

I'm totally cool if you want to make a ``subtests/docker_containers/`` (or similar name) for this now or later.  Especially if you have plans to add related tests in the future.  Though I also understand this may be speculative work, to get a feel for how Docker Autotest works, and if it would work for your needs.  No problem there.